client/app/nonComp/components/IndividualClaimHistoryTable.jsx

```diff
@@ -0,0 +1,180 @@
+
+  const ClaimCreatedFragment = () => {
+    return <React.Fragment>Claim created.</React.Fragment>;
+  };
+
+  const ClaimInProgressFragment = () => {
+    return <React.Fragment>Claim can be processed.</React.Fragment>;
+  };
+
+  const ClaimIncompleteFragment = () => {
+    return <React.Fragment>Claim cannot be processed until decision date is entered.</React.Fragment>;
+  };
+
+  const ClaimClosedFragment = (details) => {
+    return <React.Fragment>
+      Claim closed.<br />
+      <b>Claim decision date: </b>{formatDecisionDate(details.decisionDate)}
+    </React.Fragment>;
+  };
+
+  const CancelledTaskFragment = () => {
+    return <React.Fragment>
+      Claim closed.
+    </React.Fragment>;
+  };
+
+  const AddedIssueFragment = (details) => {
+    return <React.Fragment>
+      <b>Benefit type: </b>{BENEFIT_TYPES[details.benefitType]}<br />
+      <b>Issue type: </b>{details.issueType}<br />
+      <b>Issue description: </b>{details.issueDescription}<br />
+    </React.Fragment>;
+  };
+
+  const AddedIssueWithDateFragment = (details) => {
+    return <React.Fragment>
+      <AddedIssueFragment {...details} />
+      <b>Decision date: </b>{formatDecisionDate(details.decisionDate)}<br />
+    </React.Fragment>;
+  };
+
+  const AddedIssueWithNoDateFragment = (details) => {
+    return <React.Fragment>
+      <AddedIssueFragment {...details} />
+      <b>Decision date: </b>No decision date<br />
+    </React.Fragment>;
+  };
+  const CompletedDispositionFragment = (details) => {
+    return <React.Fragment>
+      <AddedIssueWithDateFragment {...details} />
+      <b>Disposition: </b>{details.disposition}<br />
+      <b>Decision description: </b>{details.decisionDescription}<br />
+    </React.Fragment>;
+  };
+
+  const RemovedIssueFragment = (details) => {
+    return <React.Fragment>
+      <AddedIssueWithDateFragment {...details} />
+      <b>Removed issue date: </b>{formatDecisionDate(details.eventDate)}
+    </React.Fragment>;
+  };
+  const WithdrewIssueFragment = (details) => {
+    return <React.Fragment>
+      <AddedIssueWithDateFragment {...details} />
+      <b>Withdrawal request date: </b>{formatDecisionDate(details.withdrawalRequestDate)}<br />
+    </React.Fragment>;
+  };
+
+  const AddedDecisionDateFragment = (details) => {
+    return <React.Fragment>
+      <AddedIssueWithDateFragment {...details} />
+    </React.Fragment>;
+  };
+
+  const DetailsFragment = (row) => {
+    let component = null;
+
+    const { readableEventType, details } = row;
+
+    details.eventDate = row.eventDate;
+
+    switch (readableEventType) {
+    case 'Claim created':
+      component = <ClaimCreatedFragment />;
+      break;
+    case 'Claim closed':
+      component = <ClaimClosedFragment {...details} />;
+      break;
+    case 'Completed disposition':
+      component = <CompletedDispositionFragment {...details} />;
+      break;
+    case 'Claim status - In progress':
```

Comment:
You might have to use the event types instead of the readable names anyhow because of cancelled (if we care since cancelled tasks shouldn't be reachable in the UI anyhow), but the P in Progress is capitalized in the readable name I think.

```suggestion
    case 'Claim status - In Progress':
```